ivy/__init__.py

```diff
@@ -843,24 +843,18 @@ def _assert_array_significant_figures_formatting(sig_figs):
 
 
 # ToDo: SF formating for complex number
-def _sf(x, sig_fig=3):
+def sf(x, sig_fig=3):
```

Comment:
let's rename this function to something verbose for code clarity, probably just `vec_sig_fig`